package.json

```diff
@@ -66,6 +66,7 @@
     "eslint": "^2.8.0",
     "fb-watchman": "^1.9.0",
     "glob": "^7.0.3",
+    "lodash": "4.13.1",
```

Comment:
Unrelated: I was recently listening to a podcast with the maintainer of lodash and he was saying that he might deprecate the individual packages in favor of one big package. Plus the one big package receives more frequent updates than the individuals and has less duplication (since it doesn't simply inline dependencies). I think we should probably switch to the main package at some point.